waffle-cli/buildTestContracts.ts

```diff
@@ -1,4 +1,6 @@
-import {compileAndSave} from '@ethereum-waffle/compiler';
+import { compileAndSave } from '@ethereum-waffle/compiler';
+import { inputToConfig } from '../waffle-compiler/src/config';
+import { generateTypes } from '../waffle-compiler/src/generateTypes';
```

Comment:
I would export `inputToConfig` and `generateTypes` in `waffle-compiler/src/index.ts` and then:
```suggestion
import { compileAndSave, inputToConfig, generateTypes } from '@ethereum-waffle/compiler';
```

Reply:
I don't think, that we want to export these two functions from src. Should it be available for waffle users? I

Reply:
I'd say why not. 

However this file is only for building test contracts and is not part of the published `src` so it's not a big deal leaving the imports this way.